packages/material-ui-lab/src/DateTimePicker/DateTimePickerTabs.tsx

```diff
@@ -9,17 +9,20 @@ import DateRangeIcon from '../internal/svg-icons/DateRange';
 import { WrapperVariantContext } from '../internal/pickers/wrappers/WrapperVariantContext';
 import { DateTimePickerView } from './shared';
 
-const viewToTabIndex = (openView: DateTimePickerView) => {
-  if (openView === 'date' || openView === 'year') {
+type TabValue = 'date' | 'time';
+
+const viewToTab = (openView: DateTimePickerView): TabValue => {
+  // TODO: what happens if `openView` is `month`?
```

Comment:
This refactoring revealed accidental coupling with the DateTimePickerTabs they handle clock and calendar view switch not the DateTimePickerView switch. Will revisit this when renaming `DayPicker` to `CalendarPicker` where `TabValue` might become `'clock' | 'calendar'`.